benchopt/tests/test_runner.py

```diff
@@ -453,3 +453,51 @@ def get_objective(self): return dict(X=0, y=0)
         assert "s" in df['p_solver_type'].unique()
         assert "s" not in df['p_dataset_type'].unique()
         assert "d" in df['p_dataset_type'].unique()
+
+
+
+def test_gpu_flag(no_debug_log):
```

Comment:
Would move this in another file than the `test_runner`, which checks for running the bench.
maybe in `test_installcnd_solver`, renamed `test_install`?